### DISCUSSION TITLE
Need help on TITLE being cropped and wrapped

### DISCUSSION THREAD

**Discussion by AndyOppelaar** [microsoft-search/pnp-modern-search#3818] [Q&A]
This should be so simple, but in the PnP resultcards long titles are being capped (...) and shortened.
How can I make sure the whole title is displayed? Can't find anything on it in the community here.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/131661619/028d6055-ddb9-4ed9-9465-91e93700d4f5)

**Comment by kasperbolarsen** [comment]
That is actually as designed in order for the card to be dynamic, and working on multiple devices and display resolutions. You can customize the Card layout if you prefer it to behave in another way :-)

**Comment by AndyOppelaar** [comment]
Yeah I get that, but how?

Met vriendelijke groet,
/Andy Oppelaar

Verstuurd vanaf mijn iPhone.


Op 20 jun 2024 om 15:23 heeft Kasper Larsen ***@***.***> het volgende geschreven:

﻿

That is actually as designed in order for the card to be dynamic, and working on multiple devices and display resolutions. You can customize the Card layout if you prefer it to behave in another way :-)

**Comment by kasperbolarsen** [comment]
https://microsoft-search.github.io/pnp-modern-search/create-custom-layouts/
would be a good starting point , and this one shows how to edit the layouts
in a interactive way,
https://www.m365thinking.com/post/work-with-your-handlebar-based-layouts-in-an-interactive-way

tors. 20. jun. 2024 18.41 skrev Andy ***@***.***>:

**Comment by AndyOppelaar** [comment]
Thanks!

Met vriendelijke groet,
/Andy Oppelaar

Verstuurd vanaf mijn iPhone.


Op 20 jun 2024 om 19:40 heeft Kasper Larsen ***@***.***> het volgende geschreven:

﻿

https://microsoft-search.github.io/pnp-modern-search/create-custom-layouts/
would be a good starting point , and this one shows how to edit the layouts
in a interactive way,
https://www.m365thinking.com/post/work-with-your-handlebar-based-layouts-in-an-interactive-way

tors. 20. jun. 2024 18.41 skrev Andy ***@***.***>:
